plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -266,6 +266,12 @@ const frontendScriptModuleBlocksToSkip = [
 	'product-filter-rating',
 	'product-filter-active',
 	'product-filter-removable-chips',
+	'add-to-cart-form',
+	'add-to-cart-with-options',
+	'add-to-cart-with-options-quantity-selector',
+	'add-to-cart-with-options-variation-selector',
+	'add-to-cart-with-options-grouped-product-selector',
+	'add-to-cart-with-options-grouped-product-selector-item',
```

Comment:
No need to do in this PR, but I wonder if we could add this info into the `blocks` const (ie, adding a property to blocks that use iAPI), so we don't need to keep this list and the one up in `webpack-config-interactivity-blocks-frontend.js` up to date.

Reply:
Also, out of curiosity, why isn't the accordion block here? :confused:

Reply:
> No need to do in this PR, but I wonder if we could add this info into the blocks const (ie, adding a property to blocks that use iAPI),

That's one way we could approach it, but I believe in future we could probably greatly simplify webpack configs and builds from how they're setup now, which is why I kept this separate. A lot of the complexity exists because of how they're managed by AssetsController but we will in future use `block.json` and I believe we'll also be able to move to a more vanilla webpack build that's closer to Gutenbergs.

Reply:
> Also, out of curiosity, why isn't the accordion block here? 😕

That looks like a mistake I made in follow up editing 🤔 Thanks I'll fix that.